Update mysql.c fix reference schema on create

7thcubic opened the pull request:
Update for schema again.

ovr commented:
`mysql.zep.c` is autogenerated file by `Zephir` language
Please fix this in `mysql.zep` file

7thcubic commented:
https://github.com/phalcon/cphalcon/pull/3213

fixed. and so i shall close this.

but damn, zephir is much easier to code den w/e the hell it is before. 

i was half guess when i fixed this previously, that concat_VSVSVSVS got me all messed up.
